### ISSUE TITLE
CNPCs stuff

### ISSUE THREAD

**Issue by yurannado** [Unknown025/Flans-Mod-Plus#170]
bullets can be blocked by NPCs' projectiles from Custom NPCs. it may cause some problems when fighting with them.Especially when you're driving a tank and trying to shoot to some MG gunners, you'll find that your shells can be exploded easily by their bullets. I think perhaps you can try to fix it.

**Comment by GoldSloth** [comment]
Hallo, thanks for the suggestion. I think this can be setup using a list or entity names for bullets to pass through, as there's already a few setup. If you can find the name of the customNPC projectile you can add it to that list when we add it.

**Comment by yurannado** [comment]
ok,i'll have a try.thx

**Comment by GoldSloth** [comment]
Tracked by https://trello.com/c/t3hfMhPe
Please reopen or new issue if needed :)
